app/models/admin_user.rb

```diff
@@ -0,0 +1,9 @@
+class AdminUser < ApplicationRecord
+  devise(
+    :database_authenticatable,
+    :recoverable,
+    :rememberable,
+    :trackable,
+    :validatable,
```

Comment:
Probably want to add 'timeoutable' and possibly 'lockable' in future? Maybe we can put in same story as logout functionality